pallets/offchain-worker/src/lib.rs

```diff
@@ -146,8 +151,9 @@ decl_module! {
 		const QuerySessionLength: u32 = T::QuerySessionLength::get();
 		const OcwQueryReward: BalanceOf<T> = T::OcwQueryReward::get();
 
+		// benchmark is 16 us in personal mac 
 		// Request offchain worker to get balance of linked external account
-		#[weight = 10_000]
+		#[weight = T::DbWeight::get().writes(1) + 16_000_000]
```

Comment:
So this is not done for accout-linker?

Reply:
already done for account linker, no recorded since it is not tested in suitable hareware.